pyccel/errors/messages.py

```diff
@@ -123,6 +123,10 @@
 INVALID_FILE_EXTENSION = 'Wrong file extension. Expecting `py` of `pyh`, but found'
 INVALID_PYTHON_SYNTAX = 'Python syntax error'
 
+# ARRAY ERRORS
+ASSIGN_ARRAYS_ONE_ANOTHER = 'Arrays which own their data cannot become views on other arrays'
+ARRAY_ALREADY_IN_USE = 'Attempt to reallocate an array which is being used by another variable'
```

Comment:
same here, `Attempt to reallocate an array that is being used by another variable`

Reply:
I don't agree with this. I've googled and found:
https://www.grammarly.com/blog/which-vs-that/

The section "Which vs. That: What’s the Difference in Usage?" agrees with my gut feeling that "which" is better